xlml/utils/aotc_reproducibility.py

```diff
@@ -0,0 +1,87 @@
+"Bash helper commands for AOTC artifacts"
+
+import os
+
+
+def set_variables_cmds():
+    set_variables = (
+        "export PROJECT=supercomputer-testing",
+        "export CLUSTER=a3plus-benchmark",
+        "export CLUSTER_REGION=australia-southeast1",
+        "NOW=$(date +%s)",
+        "export BUCKET_NAME=regression-testing-xlml",
+        "export JOB_NAME=gpt3-xlml-$NOW-175b-nemo",
+    )
+    return set_variables
+
+def set_project_commands():
+    set_project_command = (
+        "gcloud config set project $PROJECT",
+        "sudo chown -R airflow:airflow /home/airflow/composer_kube_config",
+        "gcloud container clusters get-credentials "
+        "$CLUSTER --region $CLUSTER_REGION",
+    )
+    return set_project_command
+
+def install_helm_cmds():
+    install_helm_cmd = (
+        "curl -fsSL -o get_helm.sh "
+        "https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3",
+        "chmod 700 get_helm.sh",
+        "./get_helm.sh",
+    )
+    return install_helm_cmd
+
+def namespace_cmds():
+    namespace = (
```

Comment:
We actually spent quite long time figuring this out, good job.

Can you add comments here providing contexts why this step is needed, this will help later developer understand the process better.

Reply:
done!